ts/screens/onboarding/OnboardingNotificationsPreferencesScreen.tsx

```diff
@@ -63,6 +86,38 @@ const loadingButtonProps = (): BlockButtonProps => ({
   iconColor: IOColors.bluegreyDark
 });
 
+const CustomGoBack = memo(
```

Comment:
We're using memo without actually specifying the logic to compare the condition for re-rendering the component, I guess we can avoid it since this component would rerender only if the prop `isFirstOnboarding` will change, what do you think?

Reply:
if you remove `memo`, changing reminder toggle triggers `CustomGoBack` and `Header` rerendering, so it seems to be useful